admin/templates/base.html

```diff
@@ -55,6 +55,9 @@
                 </a>
               </li>
               <li>
+                <a href="{% url 'password_change' %}" class="btn btn-flat">Change password</a>
```

Comment:
@Ghalko I chatted with Chris about this briefly - that's exactly what I was thinking, having a user-settings page for updating account information. Same conclusion - leave it as is for now but keep it noted